pyproject.toml

```diff
@@ -26,7 +26,11 @@ classifiers = [
     "Typing :: Typed",
 ]
 dynamic = ["version"]
-dependencies = ["array-api-compat>=1.1.1"]
+# dependencies = [ "array-api-compat" >= 1.1.1 ]  # DNM
+dependencies = [
+    # DNM main plus #211, #205, #207
+    "array-api-compat @ git+https://github.com/crusaderky/array-api-compat.git@9761bf2c2790416e3c117ed9f64407525b4a3498"
+]
```

Comment:
This doesn't work and I could use the help of someone more skilled in pixi-fu.
I want to test this PR (locally and in CI) against my latest changes to array-api-compat, before they are pushed into a new release.
The updated pixi.lock looks correct (see above). However, when I run `pixi run -e tests tests` I get
```
E   ImportError: cannot import name 'is_writeable_array' from 'array_api_compat' (/home/crusaderky/github/array-api-extra/.pixi/envs/tests/lib/python3.13/site-packages/array_api_compat/__init__.py)
```
`is_writeable_array` is exactly the function that has been added in the git branch temporarily referenced above.
if I look at `.pixi/envs/tests`, however, I can see that pixi installed the latest released version (1.9.1) instead.

Reply:
https://pixi.sh/latest/reference/pixi_manifest/#full-specification might help you, otherwise the pixi devs are super helpful on the prefix-dev discord

Reply:
It was a bug https://github.com/prefix-dev/pixi/issues/2677